compiler/basicTypes/Var.hs

```diff
@@ -417,7 +423,7 @@ varWeightDef = fromMaybe Omega . varWeightMaybe
 varWeight :: Id -> Mult
 varWeight id = case varWeightMaybe id of
   Just x -> x
-  Nothing -> error "Attempted to retrieve the multiplicity of a non-Id variable"
+  Nothing -> pprPanic "Attempted to retrieve the multiplicity of a non-Id variable" (ppr id)
```

Comment:
I think that the intended semantics of `pprPanic` is that the first argument is the name of the current function, and the second is the error message. So maybe it'd rather be something like

```suggestion
  Nothing -> pprPanic "varWeight" (ppr "Attempted to retrieve the multiplicity of a non-Id variable: " $$ ppr id)
```

(I never remember exactly what the pretty-printing combinators are, so I'm probably wrong)